src/dxtbx/model/detector.py

```diff
@@ -774,13 +775,21 @@ def imgCIF_H(cbf_handle, sensor):
         size = tuple(reversed(cbf_handle.get_image_size(0)))
 
         try:
-            undefined_value = find_undefined_value(cbf_handle)
+            underload = find_underload_value(cbf_handle)
+        except Exception:
+            try:
+                underload = find_undefined_value(cbf_handle) + 1
```

Comment:
Came here looking from #595. Sorry I did not comment here earlier before this was merged, but I find this line totally mysterious. Why should the "undefined value" be one less than the "underload"? This sounds like a convention, so is there something that could be pointed to by a comment?

Also, I find the terms "underload" and "overload" to be confusing. It sounds like these are outside the trusted range, whereas what is actually meant here is `min_trusted_value` and `max_trusted_value`, right?

Reply:
Agree on the first point.  I'll add some comments, over in #595.

The second one is harder, I'll also comment in #595.